modules/ROOT/pages/objc.adoc

```diff
@@ -1086,9 +1090,15 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
 
 == Release Notes
 
+===  2.6.4 -- February 2020
+
+This maintenance release includes the following important fix:
+
+- https://issues.couchbase.com/browse/CBSE-7581[CBSE-7581] Fixes memory leak, preventing crashes
```

Comment:
We do not list CBSEs in the release notes. Please use the associated CBL instead for all files in the PR.